backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -59,27 +60,41 @@ def process_row_inner(self, sample, library, container, project, parent_containe
                                individual["reference_genome"],
                                mother_obj,
                                father_obj])
+        can_use_generic_individual = (any([taxon_obj is not None,
+                                           reference_genome_obj is not None])
+                                      and not any([individual["pedigree"],
+                                                   mother_obj,
+                                                   father_obj]))
         # When the individual name is not provided any field that is stored on the individual need to raise an error.
         self.errors['individual'] = []
         self.warnings['individual'] = []
-        if not individual["name"] and need_individual:
-            if individual["taxon"]:
-                self.errors['individual'].append(f"Individual taxon requires an individual name to be provided to be saved.")
+        if not individual["name"] and need_individual and not can_use_generic_individual:
             if individual["sex"]:
-                self.errors['individual'].append(f"Individual sex requires an individual name to be provided to be saved.")
+                self.errors['individual'].append(f"Individual sex requires an individual name or taxon to be provided to be saved.")
```

Comment:
Why 'or taxon' when you haven't aren't changing for taxon? This could get in the way of refactoring :(

Reply:
There is not much to be gained by refactoring this ... The taxon here is a best guess on what was missing on the using submission ... There is also possibly the pedigree and parents messing things up but it get complex fast explaining that here ... i rely on the other error messages to carry that meaning ... If you only get that message you are likely missing a taxon... if you also have a pedigree error you should give a name to your individual ... (one error requires name or taxon the other a name). Not sure there is a simple way to get this across to the user.